cloud_foundry_api/assets/service_checks.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "agent_version": "7.21.0",
+    "integration": "Cloud Foundry API",
+    "check": "cloud_foundry_api.api.can_connect",
+    "statuses": [
+      "ok",
+      "critical"
+    ],
+    "groups": [
+      "api_url"
```

Comment:
```suggestion
      "host",
      "api_url"
```

Reply:
no host, the check should be run as a cluster check